packages/account/src/Components/personal-details/personal-details.tsx

```diff
@@ -82,8 +78,6 @@ const PersonalDetails = observer(
             traders_hub: { is_eu_user },
         } = useStore();
         const { account_status, account_settings, residence, real_account_signup_target } = props;
-        const [should_close_tooltip, setShouldCloseTooltip] = useState(false);
```

Comment:
why did we remove this ?

Reply:
Tax info has been moved from here to a separate component. Hence its not required